### ISSUE TITLE
Any method to uninstall?

### ISSUE THREAD

**Issue by juanmacio** [xflux-gui/xflux-gui#23]
Is there a method to uninstall via terminal?
Thanks.

**Comment by itsabear** [comment]
To uninstall:

```bash
sudo rm /usr/local/lib/python2.7/dist-packages/f.lux_indicator*
sudo rm -rf /usr/local/lib/python2.7/dist-packages/fluxgui
sudo rm /usr/local/lib/python2.7/dist-packages/
sudo rm -rf /usr/local/share/icons/hicolor/scalable/apps/fluxgui*
sudo rm /usr/share/applications/fluxgui.desktop
sudo rm /usr/local/bin/xflux
sudo rm /usr/local/bin/fluxgui
```

**Comment by juanmacio** [comment]
Thank you very much.
I had to correct the path to the desktop file as it was `/usr/local/share/applications/fluxgui.desktop` on my computer. Also had to remove the PPA from the Software Center.

**Comment by itsabear** [comment]
@juanmacio alright, thanks for the note. Manual install (fluxgui.desktop) should be in `/usr/local/share/applications/fluxgui.desktop`. While with ppa, it will be in `/usr/share/applications/fluxgui.desktop`.

also can you search in your $HOME directory if there any available fluxgui config files with `ls -AR | grep "flux"`?

**Comment by juanmacio** [comment]
OK, so running `ls` as above I've got: `./.gconf/apps/fluxgui`.

**Comment by itsabear** [comment]
alright..adding to the list.

**Comment by ntc2** [comment]
Thanks to @itsabear for updating the README!

**Comment by KIMWW** [comment]
Hi,
I am a newbie and need some help with this.
I am trying to uninstall f.lux (it does not work properly unfortunately). I followed [https://github.com/xflux-gui/xflux-gui/issues/23#issuecomment-246099841](url) so now f.lux does not start but when I type `ls -AR | grep "flux"` I have: fluxgui.desktop

How can I remove the left over? I tried: `sudo apt-get autoremove` but it did not find anything to remove...

**Comment by ntc2** [comment]
@KIMWW , check out https://github.com/xflux-gui/xflux-gui#ubuntudebian-manual-uninstall. Your comment link doesn't work for me.

**Comment by KIMWW** [comment]
Thanks for the reply ntc2,
I know how to uninstall f.lux but it's fluxgui that I can't uninstall completely.
I still have: fluxgui.desktop. How can I remove the left over?
The link was about the method written by Ghost (above).
> sudo rm -rf /usr/local/lib/python2.7/dist-packages/{fluxgui/,f.lux_indicator*}
sudo rm /usr/local/share/icons/hicolor/scalable/apps/fluxgui.*
sudo rm /usr/local/share/applications/fluxgui.desktop
sudo rm /usr/local/bin/{xflux,fluxgui}
rm -rf ~/.gconf/apps/fluxgui/

**Comment by ntc2** [comment]
@KIMWW, I don't understand what you mean by uninstalling f.lux as distinct from uninstalling fluxgui. While fluxgui is a GUI for flux, flux is usually downloaded and installed as part of installing fluxgui.

How did you install fluxgui?

If you installed it manually (https://github.com/xflux-gui/xflux-gui#ubuntudebian-manual-install) then you should try the manual uninstall procedure I linked to earlier (https://github.com/xflux-gui/xflux-gui#ubuntudebian-manual-uninstall), it's not exactly the same as the sequence of commands you quoted.

Of course, if you're trust trying to remove the fluxgui.desktop file, you can just `rm` it.

**Comment by KIMWW** [comment]
Sorry for not explaining clearly, English is not my native language...
Yes I installed (and uninstalled) fluxgui manually with the method you mentioned but because there was fluxgui.desktop. left  I thought it wasn't properly uninstalled. I understand now it was nothing to worry about and I can remove it with `rm` command.
Thank you for having created fluxgui, I think it's a great program, I'll try to install it again in the future because perhaps I made some mistakes when I try to install it.
Thanks for your help and for your patience.
Kim
